contracts/contracts/token/V001/BurnableToken.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.23;
+
+import "./StandardToken.sol";
+
+contract V001_BurnableToken is V001_StandardToken {
+  //
+  // Ported from OpenZeppelin's BurnableToken to use EternalStorage
```

Comment:
Hmm... I hate that we have to diverge from OpenZeppelin to use EternalStorage. We lose the confidence we would otherwise inherit from OpenZeppelin. Really opens things up to potential bugs. Not that we can't do our own audits, but I doubt we will be able to get anywhere near as many eyes on our contracts as the OpenZeppelin contracts have.